crates/aptos-dkg/src/range_proofs/univariate_range_proof.rs

```diff
@@ -493,7 +505,7 @@ where
     // Step 9: Compute d = g_1^{h(X)}
     #[cfg(feature = "range_proof_timing")]
     let start = Instant::now();
-    let d = g1_multi_exp(&pp.lagr_g1[0..num_omegas], &hh);
+    let d = VariableBaseMSM::msm(&pp.lagr_g1[0..num_omegas], &hh).expect("Failed computing msm");
```

Comment:
This is not variable base, actually.

Can you add a note to fix this later in the next PR? (This may arise in other places too.)